### PR TITLE
Guess who game 

### PR DESCRIPTION
**Pull request by matildasvensson85**

### CODE REVIEW

**Review by jesinkey — CHANGES_REQUESTED** [review]
Hej Matilda =)

Good job on almost finishing the assignment! You're so close and if you want help you can just ask and I don't mind helping you.

Screenshot 2021-02-15 at 16 55 16

The first thing I saw on the website was that the drop-down list with options was missing all of the available options for the characters (for example purple hair and so on). These should be added in the HTML under the <select id="questions">

Second was an error generated in the console when the page ran: 
Screenshot 2021-02-15 at 16 54 42
There seems to be an issue with the attribute assigned on row 307. I think the issue may be that the eventListener for selectQuestion is based off a `change` command, and if you just guess brown hair when it first pops up in the list selectQuestion isn't run, but I'm not sure. 

Just let me know if anything is unclear or you want to discuss something!

File: code/script.js [hunk 1]
```diff
@@ -254,82 +258,167 @@ const start = () => {
...
+  // board.style.display = 'flex';
+  // board.style.display = 'none';
 }
 
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
-  // This variable stores what option group (category) the question belongs to.
+  const value = questions.options[questions.selectedIndex].value;
+  console.log(value, category);
```
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply code/script.js:271]: Yes, all of the debugging console.log() statements should be removed at the end 😸

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Hannah provided very good feedback for you. You need to fallow her suggestions and fix the issues that she pointed. 
1. You need to provide more filtering options. 
2. You need to assure that al filtering functions will reorganize your board.
3. You need to provide the last page with the result of the game.
Remember that you are not alone and there are friendly people on StackOverflow and Slack ready to help 😸

**Comment by KarolCislowski** [comment]
Hello, it's your project. Do you remember me? I may be imperfect, but I believe you can still change that before you sit down to your final masterpiece!
image

**Comment by matildasvensson85** [comment]
I made the changes now :)

**Review by KarolCislowski — APPROVED** [review]
Now it works great 👍🏻

**Pull request closed without merging**